index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="headContent">
```

Comment:
@vickymarz  Please😥, my topic is about Holi festival in India if I change the background image then it's not suitable for my design topic. It will look different and the message will not be passed.

Reply:
@Kajal19-del, I do understand your concern, but the project requirement expects that you make your design look almost identical to the provided design. You are just to personalize the content of your design to make it unique ti you.

![image](https://user-images.githubusercontent.com/63473584/218028368-847ea2ed-64d4-4805-8eeb-040732eb2ff9.png)